New module to request quota increase for an AWS resource (#34)
gruntwork-io/package-terraform-utilities: modules/quota-increase/README.md

> # Request AWS Quota Increase

Reviewer: NIT: I think this should be in our standard format with Learn, Deploy, Manage sections.

Author: Where can I find a example with this? The other modules don't follow this pattern :/

Reviewer 2: Example: https://github.com/gruntwork-io/terraform-aws-eks/

# Request AWS Quota Increase

This module can be used to request a quota increase for an AWS Resource.

## Example code

See the [quota-increase example](/examples/quota-increase) for working sample code.

## Usage

Use the module in your Terraform code, replacing `<VERSION>` with the latest version from the [releases
page](https://github.com/gruntwork-io/package-terraform-utilities/releases):

```hcl
module "path" {
  source = "git::git@github.com:gruntwork-io/package-terraform-utilities.git//modules/quota-increase?ref=<VERSION>"

    request_quota_increase = {
      nat_gateway = 40,
      nacl_rules = 25
    }
}
```

The argument to pass is:

> * `request_quota_increase`: A map with the desired resource and the new quota. The current supported resources are `nat_gateway` and `nacl_rules`. Feel free to contribute to this module addin more `quota_code` and `service_code` on the [main.tf](main.tf)!

Reviewer 2: ```suggestion
* `request_quota_increase`: A map with the desired resource and the new quota. The current supported resources are `nat_gateway` and `nacl_rules`. Feel free to contribute to this module to add support for more `quota_code` and `service_code` options in [main.tf](main.tf)!
```
